tests/instruments/advantest/test_advantestQ8381.py

```diff
@@ -0,0 +1,152 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+import pytest
+from pymeasure.test import expected_protocol
+from pymeasure.instruments.advantest import AdvantestQ8381
+
+
+def test_init():
+    with expected_protocol(AdvantestQ8381, [], ):
+        pass  # Verify the expected communication.
+
+
+def test_wavelength_center():
+    with expected_protocol(
+        AdvantestQ8381,
+        [("CEN 10nm", None),
+         ("CEN?", "10")]
+    ) as inst:
+        instr = inst    # type: AdvantestQ8381
```

Comment:
Why this renaming?

You could either do "as instr:" or you could use "Inst" in all instances